### ISSUE TITLE
Could you release the code for training DenseNet and Resnet models?

### ISSUE THREAD

**Issue by yw981** [ShiyuLiang/odin-pytorch#6]
I trained a new DenseNet121 with code from [https://github.com/kuangliu/pytorch-cifar](https://github.com/kuangliu/pytorch-cifar)
`Loss: 0.179 | Acc: 95.580%`
However, it seems ODIN doesn't work on this model.
```
temperature =  200  epsilon =  0.000064
Neural network architecture:          DenseNet
In-distribution dataset:                     CIFAR-10
Out-of-distribution dataset:     Tiny-ImageNet (crop)

                          Baseline         Our Method
FPR at TPR 95%:              39.2%              27.8% 
Detection error:             11.3%              12.3%
AUROC:                       93.2%              92.2%
AUPR In:                     91.7%              87.7%
AUPR Out:                    92.0%              93.1%
```
This is the best ODIN performance I can get when "temperature =  200 and  epsilon =  0.000064" which is worse than the baseline.

Is this project related to models?
Could you release the code for training DenseNet and Resnet models in this project?
Thanks!

**Comment by YixuanLi** [comment]
Hi Tony, your epsilon seems to small compared to the optimal parameter we showed in Figure 4 (https://arxiv.org/pdf/1706.02690.pdf). There might be codebase incompatibility issue unknown. 

We have recently put together a [codebase](https://github.com/jfc43/robust-ood-detection) to reproduce all OOD detection methods (ODIN included) in one place. The DenseNet training code is also available there. You can run `python robust_ood_train.py --name Original` to get the pre-trained model. This might be the easiest solution for you. 

Please let me know if you have any other issues!
